src/js/covidMap.js

```diff
@@ -267,6 +302,32 @@ fluid.defaults("fluid.covidMap.map", {
                 return zoom < zoomThresh;
             },
             args: ["{that}.model.zoom", "{that}.options.smallMarkersBelowZoom"]
+        },
+        isMobileView: {
+            source: "isDesktopView",
+            target: "isMobileView",
+            func: value => !value
+        },
+        // Panel visibilities
+        citiesListVisibility: {
```

Comment:
This block would probably work better as a fluid.transforms.arrayToSetMembership transform, acting on the panels visibility held in both forms - e.g. visiblePanelArray and visiblePanelFlags. You can then write explicit definitions coupling all the flags - e.g.
visiblePanelFlags: {
    citiesList: "{that}.model.dom.citiesList.visible"
}
etc.